weed/command/fuse.go

```diff
@@ -0,0 +1,163 @@
+package command
+
+import (
+	"fmt"
+	"strings"
+	"strconv"
+	"time"
+	"os"
+)
+
+func init() {
+	cmdFuse.Run = runFuse // break init cycle
+}
+
+func runFuse(cmd *Command, args []string) bool {
+	argsLen := len(args)
+	options := []string{}
+
+	// at least target mount path should be passed
+	if argsLen < 1 {
+		return false
+	}
+
+	// first option is always target mount path
+	mountOptions.dir = &args[0]
+
+	// scan parameters looking for one or more -o options
+	// -o options receive parameters on format key=value[,key=value]...
+	for i := 0; i < argsLen; i++ {
+		if args[i] == "-o" && i+1 <= argsLen {
+			options = strings.Split(args[i+1], ",")
+			i++
+		}
+	}
+
+	// for each option passed with -o
+	for _, option := range options {
+		// split just first = character
+		parts := strings.SplitN(option, "=", 2)
+
+		// if doesn't key and value skip
+		if len(parts) != 2 {
+			continue
+		}
+
+		// switch key keeping "weed mount" parameters
+		switch parts[0] {
+			case "filer":
+				mountOptions.filer = &parts[1]
```

Comment:
filer now supports comma separated list of filers. Any way to make it work in this mode?

Reply:
Does it support adding a quoted filer list?

Reply:
Seems not an issue.  Double quoting is required for this case.  For example:
```
                     mount -t tmpfs none /mnt -o \
                       'context="system_u:object_r:tmp_t:s0:c127,c456",noexec'
```
https://man7.org/linux/man-pages/man8/mount.8.html

Reply:
If it works, please send a new PR to add the example to the help message.

I am going to merge this PR for now.